src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString.split(';');
+  const stylesObject = {};
+  const MIN_STYLE_LENGTH = 5;
+
+  sourceArray.forEach(el => {
+    if (el.length > MIN_STYLE_LENGTH) {
```

Comment:
interesting way to get rid of unnecessary spaces)
You can destructure your key value from each string like so
`const [key, value] = el.split(:);`
And instead of `if (el.length > MIN_STYLE_LENGTH` you could check 
```
if (key && value) {
  stylesObject[key.trim()] = value.trim();
}
```

Reply:
Thanks! Looks much better!